test/hotspot/jtreg/runtime/handshake/HandshakeTransitionTest.java

```diff
@@ -25,11 +25,13 @@
 import jdk.test.lib.Utils;
 import jdk.test.lib.process.ProcessTools;
 import jdk.test.lib.process.OutputAnalyzer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 /*
  * @test HandshakeTransitionTest
  * @summary This does a sanity test of the poll in the native wrapper.
- * @requires vm.debug
```

Comment:
Why was this deleted?
Have you tested this with release bits?

Reply:
Martin asked if we could add the diagnostics flag and make the test(s) runnable in release build.
(higher chance of a race happening in release)
So I did that, and tested it locally.